pandas/guide/source/pandas_setup.rst

```diff
@@ -79,12 +83,31 @@ TODO
 Besides the Python `.py` files, pandas source code includes C/Cython files
 which need to be compiled in order to run the development version of pandas.
 
+For Window Users: you'll need to install the compiler toolset. Depending on which version of Python you care about, you will need to choose a different download.
```

Comment:
As users are using Anaconda, I think we can assume they'll be using the latest version of Python, and simplify this. If you want, with a note at the end with the link to Python 2.7.